### PR TITLE
Derek/blank tests

### PR DESCRIPTION
**Pull request by betaBison**
Thanks for the missing fields fix. I'm pulling into your branch in hopes of speeding up pull request #101 in the main georinex repo.

This additional fixes the situation where missing fields are at the end of the entry for QZSS. This was throwing an error since the `.rstrip()` happening to count the number of fields was removing the final spaces which was resulting in `NaN` values for the final "FitIntvl" field which should instead be treated as 0.0.

I added two tests for missing fields.

### CODE REVIEW

File: src/georinex/tests/test_nav3.py [hunk 1]
```diff
@@ -258,3 +258,23 @@ def test_ionospheric_correction():
     nav = gr.load(R / "galileo3.15n")
 
     assert nav.attrs["ionospheric_corr_GAL"] == approx([0.1248e03, 0.5039, 0.2377e-01])
+
+def test_missing_fields():
+    """Tests the conditions when missing fields exist within rinex data.
+
+    """
+    nav = gr.load(R / "BRDC00IGS_R_20201360000_01D_MN.rnx", use='E', verbose=True)
+    # missing fields should be interpretted as zero and not NaN
```
Comment (jtec) [inline src/georinex/tests/test_nav3.py:267]: nit: typo

**Comment by jtec** [comment]
Looks great to me! Thanks for adding those tests!

**Review by jtec — APPROVED** [review]

**Comment by jtec** [comment]
Feel free to merge, and then we can work on getting https://github.com/geospace-code/georinex/pull/101 in ASAP.

**Comment by betaBison** [comment]
@jtec Thanks for looking at these changes. You'll have to merge these changes into your own branch since I don't have write access for your repository. Once you merge my changes in, if you'll also please change https://github.com/geospace-code/georinex/pull/101 to a pull request rather than a draft pull request, I'd be happy to comment and tag the maintainer to hopefully get the ball rolling.

**Pull request merged**